### PR TITLE
[FEAT] Setup for iOS Google Sign-in

### LINKED ISSUE #7: Setup Sign-in with Google for iOS
Since sign-in with Google has to be setup natively for each platform we need to set it up or iOS separately. After #6 is completed follow the rest of the below tutorial to add it for iOS.
* Tutorial: https://www.youtube.com/watch?v=d_Vf41Sb0v0

## Definition of done
* Page with Sign-in with Google works for iOS emulator as well as Android.

### PR DESCRIPTION
**Pull request by dhruv423**
## Overview

Implements the setup necessary to Sign-in with Google for iOS.

Closes #7 

This change is a

- [ ] Bug fix
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that will change existing functionality)
- [ ] Documentation update

## Description
Screen Shot 2023-01-08 at 5 01 21 PM
image

## Motivation/Links
To allow users with iOS devices to sign up for our app

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,7 +41,9 @@
     "@react-navigation/native-stack": "^6.9.5",
     "axios": "^1.2.1",
     "expo": "~47.0.8",
+    "expo-build-properties": "~0.4.1",
```
Comment (benjaminkostiuk) [inline package.json:44]: Do we need this? Isn't this for expo prebuild: https://docs.expo.dev/versions/latest/sdk/build-properties/

Comment (dhruv423) [reply package.json:44]: Yeah it's needed for native dependencies, I couldn't get it to build for iOS without this.
https://stackoverflow.com/questions/73415961/getting-error-when-building-ios-app-by-using-eas-expo-application-services

Comment (benjaminkostiuk) [reply package.json:44]: Makes sense 👍

**Review by benjaminkostiuk — APPROVED** [review]
Beauty!

**Pull request merged**